<?xml version="1.0" ?>
<sdf version="1.5">
    <model name="marker3">
        <!--<plugin name="name_gps" filename="libhector_gazebo_ros_gps.so">-->
            <!--<alwayson>1</alwayson>-->
            <!--<updaterate>4.0</updaterate>-->
            <!--<bodyname>basefootprint</bodyname>-->
            <!--<topicname>helipad/fix</topicname>-->
            <!--<velocitytopicname>helipad/fix_velocity</velocitytopicname>-->
            <!--<referenceLatitude>37.9707</referenceLatitude>-->
            <!--<referenceLongitude>23.7294</referenceLongitude>-->
            <!--<drift>5.0 5.0 5.0</drift>-->
            <!--<gaussiannoise>0.01 0.01 0.01</gaussiannoise>-->
            <!--<velocitydrift>0 0 0</velocitydrift>-->
            <!--<velocitygaussiannoise>0.05 0.05 0.05</velocitygaussiannoise>-->
        <!--</plugin>-->
        <link name="link">
            <gravity>0</gravity>
            <collision name="collision_marker3">
                <pose>0 0 0.0 0 0 0</pose>
                <geometry>
                    <box>
                        <size>0.5 0.5 0.01</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual_marker3">
                <pose>0 0 0.0 0 0 0</pose>
                <geometry>
                    <box>
                        <size>0.5 0.5 0.01</size>
                    </box>
                </geometry>
                <material>
                    <script>
                        <uri>model://ar_helipad/materials/scripts</uri>
                        <uri>model://ar_helipad/materials/textures</uri>
                        <name>Marker3/Diffuse</name>
                    </script>
                </material>
            </visual>
        </link>
    </model>
</sdf>
